addons/account_edi/views/account_move_views.xml

```diff
@@ -96,6 +97,12 @@
                         invisible="edi_error_count == 0 or edi_blocking_level != 'error'">
                         <div class="o_row">
                             <field name="edi_error_message" />
+                            <button name="force_cancel_edi"
+                                string="Force EDI Cancellation"
```

Comment:
```suggestion
                                string="Force Cancel"
```
I think this wording makes more sense. Indeed, this is not a cancellation of the EDI document, but rather a cancellation of the invoice _without cancelling the EDI document_.